integration_tests/src/main/python/delta_lake_delete_test.py

```diff
@@ -99,14 +99,33 @@ def write_func(spark, path):
         delete_sql="DELETE FROM delta.`{}`".format(path)
         spark.sql(delete_sql)
     with_cpu_session(setup_tables)
-    disable_conf = copy_and_update(delta_writes_enabled_conf,
-            {"spark.rapids.sql.command.DeleteCommand": "true",
-             "spark.rapids.sql.command.DeleteCommandEdge": "true",
-             "spark.databricks.delta.delete.deletionVectors.persistent": "true"})
+    disable_conf = copy_and_update(delta_delete_enabled_conf,
+        {"spark.databricks.delta.delete.deletionVectors.persistent": "true"})
 
     assert_gpu_fallback_write(write_func, read_delta_path, data_path,
                               "ExecutedCommandExec", disable_conf)
 
+@allow_non_gpu("LocalTableScanExec", *delta_meta_allow)
+@delta_lake
+@ignore_order
+@pytest.mark.parametrize("use_cdf", [True, False], ids=idfn)
+@pytest.mark.skipif(not is_databricks122_or_later(), reason="Deletion vectors are new in Databricks 12.2")
+def test_delta_deletion_vector_fallback_databricks(spark_tmp_path, use_cdf):
+    data_path = spark_tmp_path + "/DELTA_DATA"
+    def setup_tables(spark):
+        setup_dest_tables(spark, data_path,
+                          dest_table_func=lambda spark: unary_op_df(spark, int_gen),
+                          use_cdf=use_cdf, enable_deletion_vectors=True)
+    def write_func(spark, path):
+        delete_sql="DELETE FROM delta.`{}`".format(path)
+        spark.sql(delete_sql)
+    with_cpu_session(setup_tables)
+    disable_conf = copy_and_update(delta_delete_enabled_conf,
+        {"spark.databricks.delta.delete.deletionVectors.persistent": "true"})
+
+    assert_gpu_fallback_write(write_func, read_delta_path, data_path,
+                              "LocalTableScanExec", disable_conf)
```

Comment:
This doesn't seem to be the right thing to search for in a fallback.  LocalTableScanExec isn't what actually writes the deletion vector or performs the delete -- this should be looking for the CPU exec node that performs the delete, not one performing a read/scan.  There are other delete fallback tests that are passing on Databricks 12.2, curious why this wasn't modeled after those (or just reusing the non-Databricks fallback test which I would expect to work).

Reply:
So this wasn't falling back at all, which led to my confusion. I needed to implement the fallback logic for DeleteCommandEdge in addition to DeleteCommand. This is now updated.